### PR TITLE
chore: define vscode command for get logger

### PR DESCRIPTION
**Pull request by peternhale**
Establish the vscode command that will be used to fetch an instance of a logger

@W-15295896@

This work is preparation for follow-on work items, so no change in behavior.

### CODE REVIEW

File: packages/salesforcedx-vscode-core/package.nls.ja.json [hunk 1]
```diff
@@ -72,5 +72,6 @@
   "view_local_changes_text": "SFDX: ローカルの変更を表示",
   "view_remote_changes_text": "SFDX: デフォルトのスクラッチ組織内の変更を表示",
   "visualforce_generate_component_text": "SFDX: Visualforce コンポーネントを作成",
-  "visualforce_generate_page_text": "SFDX: Visualforce ページを作成"
+  "visualforce_generate_page_text": "SFDX: Visualforce ページを作成",
+  "vscode_core_services_get_logger": "salesforcedx-vscode-core ロガーサービスを取得する"
```
Comment (peternhale) [inline packages/salesforcedx-vscode-core/package.nls.ja.json:76]: translation provided by copilot, so probably needs "adjustment". This text will never be seen by a user.

File: packages/salesforcedx-vscode-core/src/index.ts [hunk 2]
```diff
@@ -385,6 +386,11 @@ const registerCommands = (
     renameLightningComponent
   );
 
+  const getCoreLoggerServiceCmd = vscode.commands.registerCommand(
```
Comment (peternhale) [inline packages/salesforcedx-vscode-core/src/index.ts:389]: register command during activation

File: packages/salesforcedx-vscode-core/src/services/getCoreLoggerService.ts [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+  }
+  error(...args: unknown[]): ILogger {
+    throw new Error('Method not implemented.');
+  }
+  fatal(...args: unknown[]): ILogger {
+    throw new Error('Method not implemented.');
+  }
+}
+
+export const getCoreLoggerService = (loggerName: string): ILogger => {
```
Comment (peternhale) [inline packages/salesforcedx-vscode-core/src/services/getCoreLoggerService.ts:68]: This function will be refined in follow-on work.

File: packages/salesforcedx-vscode-core/test/jest/services/getCoreLoggerService.test.ts [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import { LoggerLevel } from '@salesforce/vscode-service-provider';
+import {
+  CoreLoggerService,
+  getCoreLoggerService
+} from '../../../src/services/getCoreLoggerService';
+
+describe('getCoreLoggerService', () => {
```
Comment (peternhale) [inline packages/salesforcedx-vscode-core/test/jest/services/getCoreLoggerService.test.ts:13]: Tests will be refined in follow-on work

File: packages/salesforcedx-vscode-core/src/index.ts [hunk 5]
```diff
@@ -385,6 +386,11 @@ const registerCommands = (
...
+    'sf.rename.lightning.component',
```
Comment (mingxuanzhangsfdx) [inline packages/salesforcedx-vscode-core/src/index.ts:390]: shouldn't this be `"sf.vscode.core.logger.get.instance"`?

**Review by mingxuanzhangsfdx — CHANGES_REQUESTED** [review]

File: packages/salesforcedx-vscode-core/src/index.ts (hunk 5 above)
Comment (peternhale) [reply packages/salesforcedx-vscode-core/src/index.ts:390]: YES

**Review by mingxuanzhangsfdx — APPROVED** [review]
The change LGTM and there is no behavior to be observed from the change as expected. Given that there is wording added, please also involve @sbudhirajadoc to review it.

**Pull request merged**